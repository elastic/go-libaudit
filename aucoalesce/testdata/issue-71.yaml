---

tests:
  multiple_netfilter_cfg_messages: |
    type=NETFILTER_CFG msg=audit(1593786232.477:31): table=filter family=2 entries=0
    type=NETFILTER_CFG msg=audit(1593786232.477:31): table=filter family=10 entries=0
    type=SYSCALL msg=audit(1593786232.477:31): arch=c000003e syscall=272 success=yes exit=0 a0=40000000 a1=7fffd2436fb0 a2=40000040 a3=22 items=0 ppid=1 pid=769 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="(ostnamed)" exe="/usr/lib/systemd/systemd" subj=system_u:system_r:init_t:s0 key=(null)
    type=PROCTITLE msg=audit(1593786232.477:31): proctitle="(ostnamed)"
