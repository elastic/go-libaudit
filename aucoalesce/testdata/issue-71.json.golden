[
  {
    "test_name": "multiple_netfilter_cfg_messages",
    "event": {
      "@timestamp": "2020-07-03T14:23:52.477Z",
      "sequence": 31,
      "category": "configuration",
      "record_type": "netfilter_cfg",
      "result": "success",
      "session": "unset",
      "summary": {
        "actor": {
          "primary": "unset",
          "secondary": "0"
        },
        "action": "loaded-firewall-rule-to",
        "object": {
          "type": "firewall",
          "primary": "filter"
        },
        "how": "/usr/lib/systemd/systemd"
      },
      "user": {
        "ids": {
          "auid": "unset",
          "egid": "0",
          "euid": "0",
          "fsgid": "0",
          "fsuid": "0",
          "gid": "0",
          "sgid": "0",
          "suid": "0",
          "uid": "0"
        },
        "selinux": {
          "domain": "init_t",
          "level": "s0",
          "role": "system_r",
          "user": "system_u"
        }
      },
      "process": {
        "pid": "769",
        "ppid": "1",
        "title": "(ostnamed)",
        "name": "(ostnamed)",
        "exe": "/usr/lib/systemd/systemd"
      },
      "data": {
        "a0": "40000000",
        "a1": "7fffd2436fb0",
        "a2": "40000040",
        "a3": "22",
        "arch": "x86_64",
        "entries": "0",
        "exit": "0",
        "syscall": "unshare",
        "table": "filter",
        "tty": "(none)"
      },
      "netfilter_cfg_data": [
        {
          "entries": "0",
          "family": "2",
          "table": "filter"
        },
        {
          "entries": "0",
          "family": "10",
          "table": "filter"
        }
      ],
      "ecs": {
        "event": {
          "category": [
            "configuration",
            "process"
          ],
          "type": [
            "change",
            "info"
          ]
        },
        "user": {
          "effective": {},
          "target": {},
          "changes": {}
        },
        "group": {}
      }
    },
    "warnings": [
      "duplicate key (family) from NETFILTER_CFG message"
    ]
  }
]
